daemon/main.go

```diff
@@ -53,6 +80,12 @@ func main() {
 	log := ctrl.Log.WithName("Daemon Init")
 	log.Info("Daemon init")
 
+	err = copyCNI()
```

Comment:
Unfortunately due to the delta between RHEL versions we should be using the `entrypoint.sh` method to copy the CNI. Same code for SRIO-CNI and Multus:

https://github.com/openshift/sriov-cni/pull/100#issuecomment-2009001716

Reply:
+1

Reply:
Can you elaborate on why we need to do the copy outside of the daemon?

Reply:
Took a note that we will need to address this. More detail on this: cp from rhel 9 can be used on rhel 8, but we can't use our go binary if it is running on rhel 9 container while the host system is still rhel 8